mllib/src/main/scala/org/apache/spark/ml/tree/treeParams.scala

```diff
@@ -360,7 +362,8 @@ private[ml] trait RandomForestParams extends TreeEnsembleParams {
     "The number of features to consider for splits at each tree node." +
       s" Supported options: ${RandomForestParams.supportedFeatureSubsetStrategies.mkString(", ")}",
     (value: String) =>
-      RandomForestParams.supportedFeatureSubsetStrategies.contains(value.toLowerCase))
+      RandomForestParams.supportedFeatureSubsetStrategies.contains(value.toLowerCase)
+      || value.matches("^(?:[1-9]\\d*|0\\.\\d*[1-9]\\d*\\d*|1\\.0+)$"))
```

Comment:
I was able to remove the last `\\d*` and still match `"0.0250". Since `\\d*` matches 0 or more occurrences of a digit [0-9], I don't see why there needs to be two consecutively.